.eslintrc.json

```diff
@@ -38,8 +38,7 @@
         "asyncArrow": "always"
       }
     ],
-    "testing-library/await-async-query": "error",
-    "testing-library/no-await-sync-query": "error",
```

Comment:
should we add this rule: no-await-sync-queries?

Reply:
`no-await-sync-queries` doesn’t seem to exist in their code even though their documentation mentions it.